accepted/2024/support-nuget-authentication-plugins-dotnet-tools.md

```diff
@@ -0,0 +1,215 @@
+# ***Support for NuGet authentication plugins deployed via .NET tools***
+
+- Author Name: https://github.com/kartheekp-ms
+- GitHub Issue: https://github.com/NuGet/Home/issues/12567
+
+## Summary
+
+Currently, NuGet utilizes a [cross-platform plugin model](https://learn.microsoft.com/nuget/reference/extensibility/nuget-cross-platform-plugins#supported-operations) which is primarily used for [authentication against private feeds](https://learn.microsoft.com/en-us/nuget/reference/extensibility/nuget-cross-platform-authentication-plugin). It also supports the [package download](https://github.com/NuGet/Home/wiki/NuGet-Package-Download-Plugin) operation.
```

Comment:
Remove `/en-us/` from learn.microsoft.com link

Reply:
Good catch. Committed changes locally, will be part of next push.

Reply:
Addressed the feedback.